backend/prismaCalls.ts

```diff
@@ -235,13 +235,31 @@ export async function checkMetricsExist(metadataId: string): Promise<boolean> {
 
 export async function getDownloadCount(packageId: string): Promise<number> {
     const downloadEntries = await prisma.packageHistoryEntry.findMany({
-      where: {
-        metadata: {
-          id: packageId, // Filtering by the package ID
-        },
+        where: {
+            metadata: {
+            id: packageId, // Filtering by the package ID
+            },
         action: Action.DOWNLOAD, // Filtering by the action "DOWNLOAD"
-      },
+        },
     });
-  
+
     return downloadEntries.length; // Return the length of the filtered entries
-  }
\ No newline at end of file
+}
+
+
+export async function resetDatabase() {
+    try {
+        await prisma.package.deleteMany();
```

Comment:
Can you do this by getting all the models and then going through each one and deleting them without having to know all the models before hand? I think it'd make it easier that way in case we have to add a model or something in the future.

Reply:
Everything else is good